### PR TITLE
Struct page renamed, and linking to previous chapters

### PR DESCRIPTION
**Pull request by eksperimental**
plus a few minor touch-ups

### CODE REVIEW

**Comment by josevalim** [comment]
Let's please leave the docs, tests one as is. It is about docs, it is about tests, and also doctests. :)

**Comment by eksperimental** [comment]
alright... fixed :-p

**Comment by eksperimental** [comment]
bump! :-)

File: getting-started/protocols.markdown [hunk 1]
```diff
@@ -87,7 +87,7 @@ iex> Blank.blank?("hello")
 
 The power of Elixir's extensibility comes when protocols and structs are used together.
 
-In the previous chapter, we have learned that although structs are maps, they do not share protocol implementations with maps. Let's define a `User` struct as in the previous chapter:
+In the [previous chapter](/getting-started/struct.html), we have learned that although structs are maps, they do not share protocol implementations with maps. Let's define a `User` struct as in that chapter:
```
Comment (josevalim) [inline getting-started/protocols.markdown]: Shouldn't this be `structs.html`? :)

**Comment by eksperimental** [comment]
done!, one question jose,, do I need to let you know when I add a commit to a pull request, or you get notified?
and if so. do you also get notified when I do a "push -f" ?

**Comment by ericmj** [comment]
@eksperimental You need to add a comment, we do not get notified on pushed.

**Comment by eksperimental** [comment]
thanks for the explanation  @ericmj

**Pull request merged**